pkgs/by-name/at/atuin/package.nix

```
@@ -19,12 +20,28 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-fuVSn1vhKn2+Tw5f6zBYHFW3QSL4eisZ6d5pxsj5hh4=";
   };
 
+  patches = [
+    # atuin with bash-preexec wasn't recording history properly after searching,
+    # backport recent fix until next release
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/atuinsh/atuin/pull/1729.patch";
+      sha256 = "sha256-cG99aLKs5msatT7vXiX9Rn5xur2WUjQ/U33nOxuon7I=";
```

```suggestion
    # backport recent fix until next release
    # https://github.com/atuinsh/atuin/pull/1729
    (fetchpatch {
      url = "https://github.com/atuinsh/atuin/commit/cb11af25afddbad552d337a9c82e74ac4302feca.patch.patch";
      sha256 = "sha256-cG99aLKs5msatT7vXiX9Rn5xur2WUjQ/U33nOxuon7I=";
```
lets use a nicer URL